container-test/pom.xml

```diff
@@ -38,10 +38,6 @@
       <groupId>commons-collections</groupId>
       <artifactId>commons-collections</artifactId>
     </dependency>
-    <dependency>
```

Comment:
Please verify with 'mvn dependency:tree'

Reply:
It's not there 👍